omnibus/config/software/datadog-a7.rb

```diff
@@ -0,0 +1,36 @@
+name "datadog-a7"
```

Comment:
this def is very specific to the Agent, no need to keep it in `omnibus-software`